content/migration.md

```diff
@@ -8,6 +8,16 @@ To learn more about the new features we've added in the v8, check out this [link
 The `HttpModule` exported from the `@nestjs/common` package has been deprecated and will be removed in the next major release.
 Instead, please use the `@nestjs/axios` package (otherwise, there are no API differences).
 
+#### Terminus
+
+##### HttpHealthIndicator
+
+`HttpHealthIndicator` requires `@nestjs/axios` to be installed as well as `HttpModule` to be imported. (otherwise, there are no API differences)
```

Comment:
```suggestion
`HttpHealthIndicator` requires `@nestjs/axios` to be installed as well as `HttpModule` to be imported. Otherwise, there are no API differences.
```